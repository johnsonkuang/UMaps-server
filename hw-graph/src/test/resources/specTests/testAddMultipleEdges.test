# Test Add Multiple Edges
# tests the creation of multiple edge creations

# Create a graph
CreateGraph graph1

# Add four nodes to the graph
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4

# Add a edge connecting n1 and n2
AddEdge graph1 n1 n2 A

# Add a edge connecting n2 and n3
AddEdge graph1 n2 n3 B

# Add a edge connecting n2 and n4
AddEdge graph1 n3 n4 C

# Add a edge connecting n4 and n1
AddEdge graph1 n4 n1 D