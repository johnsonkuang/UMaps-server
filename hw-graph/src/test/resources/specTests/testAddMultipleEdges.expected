# Test Add Multiple Edges
# tests the creation of multiple edge creations

# Create a graph
created graph graph1

# Add four nodes to the graph
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1

# Add a edge connecting n1 and n2
added edge A from n1 to n2 in graph1

# Add a edge connecting n2 and n3
added edge B from n2 to n3 in graph1

# Add a edge connecting n2 and n4
added edge C from n3 to n4 in graph1

# Add a edge connecting n4 and n1
added edge D from n4 to n1 in graph1